### PR TITLE
Fix CustomApp crash on launch

### LINKED ISSUE #2285: CustomApp Crashes on launch (when dev is merged with feature/nav)
The described issue is observed when `develop` is merged with the changes proposed from `feature/navigation`
_CustomApp_
The custom app seems to be jammed completely.
* It crashes on its first install(please reinstall the app after the uninstallation as previous settings may not lead it to crash) with a border of red lines over the screen showing that there was an attempt made to launch the app and it disappears after the crash occurs leaving user with the permission request for Media access.
* After this when app is next launched(not on reinstallation) it says: (umm..., let me attach the exact visuals) [Here they are :link: ](https://drive.google.com/file/d/18aXlGGoN76Ox2EsJWbT9B-pBnkWobVDf/view?usp=sharing)
* Repeating the process causes no changes but crashes

### PR DESCRIPTION
**Pull request by s-ayush2903**
Close #2285 

**Changes made:** 
* Added null check for array size being passed for opening tabs. See commit  [#2285 Fixed CustomApp crash on launch](https://github.com/kiwix/kiwix-android/commit/8885620d45d6c7daf1e157a89a25c635466b9492) 
* Other changes are due to the recent translateWiki merge in the develop and branch syncs

**Visuals:**
[Link :link: ](https://drive.google.com/file/d/1N2jC6fa64rEkzCUYxY_Bmx7YTj5nDC75/view?usp=sharing)
Links are being attached due to the poor resolutions of gifs that were being attached previously. If there is any problem in attaching links or some preference for gifs, i'll be following the same

### CODE REVIEW

File: custom/src/main/java/org/kiwix/kiwixmobile/custom/main/CustomReaderFragment.kt [hunk 1]
```diff
@@ -107,9 +107,10 @@ class CustomReaderFragment : CoreReaderFragment() {
     val urls = JSONArray(zimArticles)
     val zimPositions = JSONArray(settings.getString(TAG_CURRENT_POSITIONS, null))
     selectTab(currentTab)
-    loadUrlWithCurrentWebview(UpdateUtils.reformatProviderUrl(urls.getString(currentTab)))
-    getCurrentWebView().scrollY = zimPositions.getInt(currentTab)
-  }
```
Comment (Frans-Lukas) [inline custom/src/main/java/org/kiwix/kiwixmobile/custom/main/CustomReaderFragment.kt:112]: Accidental removal of bracket.

e: /Users/runner/work/kiwix-android/kiwix-android/custom/src/main/java/org/kiwix/kiwixmobile/custom/main/CustomReaderFragment.kt: (236, 2): Missing '}'

Comment (s-ayush2903) [reply custom/src/main/java/org/kiwix/kiwixmobile/custom/main/CustomReaderFragment.kt:112]: Considered! thanks for help :-)

**Comment by JensKorte** [comment]
My kiwix from F-Droid crashed, too. With https://download.kiwix.org/nightly/2020-08-18/dev-universal-debug.apk  I have no problem.

**Comment by macgills** [comment]
@JensKorte we haven't updated our FDroid release in a long time, this is unrelated to your problem I would say

**Pull request merged**